build_overrides/nxp_sdk.gni

```diff
@@ -14,5 +14,9 @@
 
 declare_args() {
   # Root directory for NXP SDKs.
-  nxp_sdk_build_root = "//third_party/nxp"
+  nxp_sdk_matter_support_root = "//third_party/nxp/nxp_matter_support"
+}
+
+declare_args() {
```

Comment:
could we merge this in a single declare args? saves some vertical space.

Reply:
@andy31415  I got a GN error which recommend to separate them in 2 declare_args (as below):

2024-06-20 09:40:44 INFO    ERROR at //build_overrides/nxp_sdk.gni:19:27: Reading a variable defined in the same declare_args() call.
2024-06-20 09:40:44 INFO    
2024-06-20 09:40:44 INFO    If you need to set the value of one arg based on another, put
2024-06-20 09:40:44 INFO    them in two separate declare_args() calls, one after the other.

Do you have other recommendation to fix this error ?